parser.py

```diff
@@ -9,23 +9,52 @@
 import sys
 from os import listdir
 from os.path import isfile, join
-nltk.download('punkt')
+from collections import defaultdict
+import json
 
 def GetLanguageId(language):
-    language_id_list = logic.SQLQuery("select Lang_ID from lang_ref where Lang_Desc = '"+ language +"'")
+    language_id_list = logic.SQLQuery("select Lang_ID from lang_ref where Lang_Desc = '" + language + "'")
 
-    # Condition for language not present in database
     if len(language_id_list) == 0:
         return -1
     else:
         return language_id_list[0]
 
+def StoreIndexing(dictionary):
+    f = open("indexing.txt", "w")
+    f.write(dictionary)
+    f.close()
+
+def CreateIndexing(sentence, line_id, dictionary):
+    word_list = sentence.split()
+    # number of parts formed for each tagged word in the corpus(word/pos)
+    tagged_word_parts = 2   
+    
+    for word in word_list:
+        # skip untagged words
+        if(len(word.split('/')) != tagged_word_parts):
+            continue
+
+        subtag = (word.split('/')[1]).upper()
+        # ignore -TL suffix in subtags since they are just to indicate that the word occurs in title
```

Comment:
good naming in this area, could we just fix the formatting of the lines. It can sometimes be hard to follow

Reply:
Do you mean removing blank lines by formatting of lines ?

Reply:
Yes

Reply:
I think the formatting is okay, it's just the comments that are no looking formatted, it can be an issue of Git.